polyfill/lib/ecmascript.mjs

```diff
@@ -1631,9 +1633,14 @@ export const ES = ObjectAssign({}, ES2019, {
     return new TemporalTimeZone(ES.TemporalTimeZoneFromString(fmt.resolvedOptions().timeZone));
   },
   ComparisonResult: (value) => (value < 0 ? -1 : value > 0 ? 1 : value),
+  NormalizeOptionsObject: (options) => {
+    if (options === undefined) return {};
+    if (options !== null && (typeof options === 'object' || typeof options === 'function')) {
+      return options;
+    }
```

Comment:
```suggestion
    if (ES.Type(options) === 'Object') {
      return options;
    }
```

Reply:
Thanks for the suggestion, that makes things easier to follow.